.evergreen.yml

```diff
@@ -368,6 +368,23 @@ tasks:
           image_type: readiness-probe-init
           release: true
 
+  - name: release_agent_ubuntu
+    commands:
+      - func: clone
+      - func: setup_virtualenv
+      - func: build_and_push_image_sonar
+        vars:
+          image_type: agent-ubuntu
+          release: true
+
+  - name: release_agent_ubi
+    commands:
+      - func: clone
+      - func: setup_virtualenv
+      - func: build_and_push_image_sonar
+        vars:
+          image_type: agent-ubi
+          release: true
```

Comment:
[q] I guess there's no way to have a single task and somehow pass to it the `image_type` instead of duplicating this?

Reply:
I think this is most re-use we can get, the main thing is the re-use of `build_and_push_image_sonar`